setup.py

```diff
@@ -3,29 +3,38 @@
 install_requires=[
     'flask-restful>=0.3.0',
     'flask-sqlalchemy >= 2.3.0',
-    'flask',
+    'sqlalchemy_utils>=0.30',
+    'flask>=1.0.0',
     'sqlalchemy>=1.1',
 ]
 
+extras_require = {
+    "docs": ["sphinx >= 1.4", "sphinx_rtd_theme", "sphinx-autodoc-typehints"],
```

Comment:
`typehints` is being used? I ask because I don't see it declared in `conf.py`

Reply:
removed